src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ValidationService.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.crowncommercial.dts.scale.cat.service;
+
+import org.springframework.stereotype.Service;
+import lombok.RequiredArgsConstructor;
+import uk.gov.crowncommercial.dts.scale.cat.exception.ResourceNotFoundException;
+import uk.gov.crowncommercial.dts.scale.cat.model.OCID;
+import uk.gov.crowncommercial.dts.scale.cat.model.entity.ProcurementEvent;
+import uk.gov.crowncommercial.dts.scale.cat.repo.RetryableTendersDBDelegate;
+
+/**
+ * Captures Project and Event input validation functionality
+ */
+@Service
+@RequiredArgsConstructor
+public class ValidationService {
+
+  private final RetryableTendersDBDelegate retryableTendersDBDelegate;
+
+  /**
+   * Validate the project and event IDs and return the {@link ProcurementEvent} entity
+   *
+   * @param projectId
+   * @param eventId
+   * @return procurement event entity
+   */
+  public ProcurementEvent validateProjectAndEventIds(final Integer projectId,
```

Comment:
Nice - have virtually the exact same block of code refactored out in SCC-446 (just not in a separate class) - will piggy back off these changes in that ticket too